compiler/x/codegen/FPTreeEvaluator.cpp

```diff
@@ -722,6 +722,19 @@ TR::Register *OMR::X86::TreeEvaluator::fpSqrtEvaluator(TR::Node *node, TR::CodeG
    return result;
    }
 
+TR::Register *OMR::X86::TreeEvaluator::dsqrtEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::Node *operand = node->getFirstChild();
+   TR::Register *opRegister = cg->evaluate(operand);
+   TR::Register *targetRegister = (operand->getReferenceCount()==1) ? opRegister : cg->allocateRegister(TR_FPR);
```

Comment:
I don't think we need to check reference count here. Local RA should handle this perfectly well as the register containing `operand` will be dead following it's last (and only) use within the `SQRTSDRegReg` instruction. We should just replace this with

```
TR::Register *targetRegister = cg->allocateRegister(TR_FPR);
```

similar to how it is on Power.